client/src/components/Navbar/Navbar.jsx

```diff
@@ -1,53 +1,29 @@
 import React from 'react';
+import './Navbar.css';
 import { AppBar, Toolbar, Typography, Button } from '@mui/material';
-import { useTheme } from '@mui/material/styles'; // Import the useTheme hook
-import { makeStyles } from '@mui/styles';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1,
-  },
-  title: {
-    flexGrow: 1,
-  },
-  button: {
-    border: '1px solid transparent',
-    marginLeft: theme.spacing(2),
-  },
-
-  buttonHover: {
-    '&:hover': {
-      backgroundColor: '#8966f1',
-      transition: 'background-color 0.5s, border 0.5s',
-    },
-  },
-}));
+////////// Matrial UI ///////////
 
 const Navbar = () => {
-  const theme = useTheme();
-  const classes = useStyles();
   return (
-    <div className={classes.root}>
-      <AppBar position='static' color='primary'>
+    <div>
+      <AppBar
+        position='static'
+        sx={{ background: '#212529', paddingBottom: '15px' }}
```

Comment:
You can add some more `paddingTop: 15px` also that should be even out and it looks also be improved.